infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -141,280 +158,315 @@ private boolean isBootstrapped(CorfuMsg msg, ChannelHandlerContext ctx, IServerR
         return true;
     }
 
+    private boolean isBootstrapped(HeaderMsg reqHeader, ChannelHandlerContext ctx, IServerRouter r) {
```

Comment:
The old implementation also logs a message. Can you put that back, but as a debug log?